model/participants.go

```diff
@@ -25,6 +25,8 @@ var (
 	RoleRemixer     = Role{"remixer"}
 	RoleDJMixer     = Role{"djmixer"}
 	RolePerformer   = Role{"performer"}
+	// Credit where the artist is an album artist or artist
+	RolePrimary = Role{"primary"}
```

Comment:
I'm not sure about the "primary" name, but can't come up with a better one.

What do you think about RoleMainCredit (maincredit)?